homeassistant/components/proxmoxve/__init__.py

```diff
@@ -306,7 +325,7 @@ def __init__(self, host, port, user, realm, password, verify_ssl):
         self._proxmox = None
```

Comment:
This conflicts with the type hint in `get_api_client`
If it can be `None`, then you should adjust it with `get_api_client() -> ProxmoxAPI | None:`
If it should never be `None`, then you should define the type hint directly at the class level instead:
```python
class ProxmoxClient:
    """A wrapper for the proxmoxer ProxmoxAPI client."""

    _proxmox: ProxmoxAPI

    def __init__(
```